Makefile

```diff
@@ -27,6 +27,10 @@ test-e2e: build
 test-e2e-release: build
 	PASSES="release e2e" ./scripts/test.sh
 
+.PHONY: fuzz
+fuzz:
+	PASSES="fuzz" ./scripts/fuzzing.sh
```

Comment:
Could `make fuzz` be self sufficient to call without additional arguments (TARGET_PATH, FUZZ_TARGET)?